src/modules/event/mutations/EventUpdateMutation.ts

```diff
@@ -0,0 +1,78 @@
+import { GraphQLNonNull, GraphQLID, GraphQLString, GraphQLBoolean } from 'graphql';
+
+import { mutationWithClientMutationId } from 'graphql-relay';
+
+import { errorField, getObjectId, successField } from "@entria/graphql-mongo-helpers";
+
+import EventModel from '../EventModel';
+
+import * as EventLoader from '../EventLoader';
+
+import EventType from '../EventType';
+
+const mutation = mutationWithClientMutationId({
+  name: 'EventUpdate',
+  description: "Update a Event",
+  inputFields: {
+    eventId: {
+      type: new GraphQLNonNull(GraphQLID),
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    start: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    end: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    allDay: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+    },
+  },
+  mutateAndGetPayload: async ({ eventId, name, start, end, allDay }) => {
+    const event = await EventModel.findById({
+      _id: getObjectId(eventId), 
+      eventId,
+      name, 
+      start, 
+      end, 
+      allDay
+    });
+
+    if (!event) {
+      return {
+        error: 'Event not found',
+      };
+    }
+
+    await EventModel.updateOne({
```

Comment:
this fn has two args, the first one is the condition (only needs the _id match) the second one is the update query (use $set https://www.mongodb.com/docs/manual/reference/operator/update/set/)